atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -95,7 +97,7 @@ public DbWriteTable createWrite(TableReference tableRef, ConnectionSupplier conn
         TableValueStyle tableValueStyle = valueStyleCache.getTableType(conns, tableRef, config.metadataTable());
         switch (tableValueStyle) {
             case OVERFLOW:
-                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, tableRef);
+                return OracleOverflowWriteTable.create(config, conns, oracleTableNameGetter, oraclePrefixedTableNames, tableRef);
```

Comment:
I think it makes sense to store it in a field to be consistent with how this class is used in DbKvs (i.e. make it long-lived everywhere rather that short-lived here and long-lived in DbKvs).